src/EFCore.MySql/EFCore.MySql.csproj

```diff
@@ -17,6 +17,7 @@
     <PackageReference Include="Microsoft.EntityFrameworkCore.Relational" Version="$(MicrosoftEntityFrameworkCoreRelationalVersion)" />
     <PackageReference Include="MySqlConnector" Version="$(MySqlConnectorVersion)" />
     <PackageReference Include="Pomelo.JsonObject" Version="$(PomeloJsonObjectVersion)" />
+    <PackageReference Include="Castle.Core" Version="$(CastleCoreVersion)" />

```

Comment:
I see that `Castle.Core` was added as a dependency between 2.2.1 and 2.2.6; is this necessary?

Reply:
Added a comment here, where the dependency was added: https://github.com/PomeloFoundation/Pomelo.EntityFrameworkCore.MySql/commit/74bf3a90728a2c7998a4ff780d0d97e65a7e3be2#r35379779

Reply:
I totally missed that!